src/main/java/uk/gov/pay/directdebit/mandate/api/DirectDebitInfoFrontendResponse.java

```diff
@@ -21,96 +21,11 @@
 @JsonFormat(shape = JsonFormat.Shape.OBJECT)
```

Comment:
I restructured this class to bring fields to the top and static classes to the bottom to make it easier to read what this class as a response looks like.